File: Arrays/Easy/Python/2011. Final Value of Variable After Performing Operations.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        ans = 0

        for s in operations:
            if s[0] == '-' or s[len(s) - 1] == '-':
                ans -= 1
            else:
                ans += 1
        
        return ans
        

File: Arrays/Easy/C++/2011. Final Value of Variable After Performing Operations.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        
        int X = 0;

        for(int i = 0; i < operations.size(); i++){
            string str = operations[i];
            if(str.at(0) == '+' || str.at(2) == '+'){
                X += 1;
            }
            else{
                X -= 1;
            }
        }

        return X;
    }
};
